app/com/arpnetworking/database/h2/triggers/AlertsUpdateEtagTrigger.java

```diff
@@ -26,6 +26,6 @@
      * Public no args constructor.
      */
     public AlertsUpdateEtagTrigger() {
-        super("portal.alerts_etag_seq");
+        super("portal.alerts_etags", 15);
```

Comment:
Sorry, I completely missed the classpath on this. That's fine. 

We really need to drop H2 and use Postgres via Docker. Unfortunately, as I recall you said SBT does not have a good Docker plugin ... and I doubt you like what I'm thinking of doing ;)